### ISSUE TITLE
[bug] Favorites over third are showing 0 chapters after update

### ISSUE THREAD

**Issue by molitar** [dazedcat19/FMD2#108]
**Prerequisite**
- [x ] I'm using the latest version.
- [ ] All website modules are up to date.

- [ ] This is not temporary server issue.

**System information**
Windows 10 64 bit

**Describe the bug**
Checking for chapters in favorites has over 70+ failures every day since this latest version.  I'm going to hunt down the previous release I didn't have a single chapter fail to check before this release. 

**To Reproduce**
Have over 300+ manga in favorites and check favorites. over a third fail to check with orange error 0 chapters. I right click and manually check chapters again and it will update. 

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by molitar** [comment]
Further info on this bug.  I tried these steps.

1. Manually checked chapters of about 30 0 chapters at a time. Chapters updated.
2. Closed FMD 2 and than restarted. All the previous chapters showing 0 showed 0 again. 
3. Repeat step 1 than shut down.
4. Downgrade to FMD 2 previous release .23
5. Started FMD 2 and sure enough all the previous chapters showed 0 again
6. In Favorites used the check favorite button again. All chapters check fine. 
7. Shut down restarted and all chapters still showed properly

It seems that for some reason .24 was not actually updating the databases with the chapter updates.

**Comment by dazedcat19** [comment]
I can't understand your problem. Was the new chapter check fine but not stored?
I did check it. I didn't have any issue.

@Slasar41 can you please check it?

**Comment by Slasar41** [comment]
I tested the latest version on two different PC that have different series in favorites since before update and both doesn't have any issue.

@molitar Maybe upload your complete latest FMD installation here so we can test it.

**Comment by molitar** [comment]
Ok I had a chance to update again and before I did there was no chapters showing 0.  I updated than ran a check favorites and immediately had around 5 chapters showing orange.  I immediately zipped upped the entire directory except for backup and uploading as requested.

As files is quite large I uploaded to Google Drive.

https://drive.google.com/file/d/1W-okLDpG6I6jLTgwdCF9o1ubHk-FaDAd/view?usp=sharing
